en/starter/settingup.md

```diff
@@ -24,7 +24,7 @@ redirect_from: "/starter/setting up.html"
 <hr>
 ## Ubuntu Linux
 
-Kitura is tested on Ubuntu 14.04 LTS, Ubuntu 15.10, and Ubuntu 16.04 LTS.
+Kitura is tested on Ubuntu 14.04 LTS.
```

Comment:
Kitura is tested on Ubuntu 16.04 LTS locally by developers and soon in our Travis-CI toolchain